scripts/bootstrap-aio.sh

```diff
@@ -51,7 +51,7 @@ if apt_artifacts_available; then
   export RPCO_APT_ARTIFACTS_AVAILABLE="yes"
 else
   export RPCO_APT_ARTIFACTS_AVAILABLE="no"
-  rm -f ${BASE_DIR}/group_vars/all/apt.yml
```

Comment:
This intentionally dirties the git tree - only for an AIO, and only when apt artifacts aren't available. It would only ever happen in the gate, so better to leave as-is.

Reply:
done.

Reply:
if the group_vars are in the "/etc/openstack_deploy" do we still need this ?

Reply:
Yes, because we want to prevent them from getting there in gate tests where the apt artifacts do not yet exist for a release version. This is important for PR testing where the release version is changing.

Reply:
Why was this removed again? It needs to be there to cater for tests where the container artifacts for a release do not exist when building an AIO. That's why the tests failed.

Reply:
done.